examples/rbfe_single.py

```diff
@@ -71,10 +72,17 @@ def run_epoch(ff, mol_a, mol_b, core):
         # solvent leg
         host_args = []
         for lambda_idx, lamb in enumerate(lambda_schedule):
-            gpu_idx = lambda_idx % cmd_args.num_gpus
-            host_args.append((gpu_idx, lamb, host_system, minimized_host_coords, host_box, cmd_args.num_equil_steps, cmd_args.num_prod_steps))
-        
-        results = pool.map(functools.partial(wrap_method, fn=rfe.host_edge), host_args, chunksize=1)
+            host_args.append((lamb, host_system, minimized_host_coords, host_box, cmd_args.num_equil_steps, cmd_args.num_prod_steps))
```

Comment:
Last nitpick: we should simplify this into:

``` python
futures = []
for lambda_idx, lamb in enumerate(lambda_schedule):
    futures.append(client.submit(rfe.host_edge, lamb, host_system, minimized_host_coords, host_box, cmd_args.num_equil_steps, cmd_args.num_prod_steps))
```

Reply:
Reflected in https://github.com/proteneer/timemachine/pull/325/commits/b0374bd313e34de2246359efbe8273e316735d6f